src/engraving/io/msczwriter.h

```diff
@@ -44,21 +44,32 @@ class MsczWriter
     void close();
     bool isOpened() const;
 
-    void writeScore(const QByteArray& data);
-    void writeThumbnail(const QByteArray& data);
-    void addImage(const QString& fileName, const QByteArray& data);
-    void writeAudio(const QByteArray& data);
+    void writeScoreFile(const QByteArray& data);
+    void writeThumbnailFile(const QByteArray& data);
+    void addImageFile(const QString& fileName, const QByteArray& data);
+    void writeAudioFile(const QByteArray& data);
+    void writeAudioSettingsJsonFile(const QByteArray& data);
 
 private:
 
     struct Meta {
-        QString mscxFileName;
-        std::vector<QString> imageFilePaths;
-        QString audioFile;
-
+        std::vector<QString> files;

```

Comment:
might make sense to use io::paths

Reply:
not now.
At the moment engraving module, not dependent on anything, and this is good. `io:path` in another module, and it depends on a lot of things